src/docs/product/issues/issue-details/performance-issues/slow-db-queries/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Slow DB Queries'
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/slow-db-queries/
+description: 'Learn more about Slow DB Query issues and how to diagnose and fix them.'
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Slow DB Query issues are created when a particular `SELECT` SQL query in your application consistently takes longer than `1000ms` to resolve.
+
+## Detection Criteria
+
+The detector for this performance issue looks at singular database spans with the following criteria:
+
+- The span operation starts with `db` and the query is stored within the span description
+- The query is specifically a `SELECT` query
+- The span duration is at least `1000ms`
+
+Keep in mind that an issue will not be created unless the query in question is _consistently_ taking longer than `1000ms`. More specifically, the issue is created only if the same query is detected as a slow query
+in 100 events within a 24 hour period.
```

Comment:
I'm a bit reluctant to put the noise thresholds in the docs but we're probably better off being explicit. I think if you do include it you should mention that it's collected only once the `100` event threshold is surpassed, since that might be surprising. This also means that sometimes the issue won't correlate as well to a release since the `100` threshold could delay the first issue creation.

Reply:
That's true, although we have plans of fixing this so we include all previously collected events once the threshold is passed